### ISSUE TITLE
mime types

### ISSUE THREAD

**Issue by mazondo** [mazondo/harpoon#2]
Set mime types appropriately when pushing files

**Comment by makenosound** [comment]
I’m running into this problem at the moment — we’re deploying some legacy content with Harpoon and older versions of Internet Explorer will not parse the CSS/JS without the correct mime-types set.

Any chance of adding support?

**Comment by mazondo** [comment]
Is this for Amazon s3?  It should be setting the mine type correctly if so. I can take a look and see what I find.

**Comment by makenosound** [comment]
Yep! I'm seeing no mime-types at all for a project I'm pushing up to Amazon S3.
